lib/ts/station/fsm_behaviour.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Station.FsmBehaviour do
```

Comment:
Station.FSMBehavior

Reply:
Similar to Station.FSM the module will then be named f_s_m_behaviour.

Reply:
well. FSM is an abbreviation - not a regular work. As per camel case convention, we are allowed to keep all capitals even in the file name.

Reply:
Ok. then we'll have FSM.ex and FSM_behaviour.ex.